website_and_docs/content/documentation/selenium_manager.en.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Selenium Manager (Beta)"
+linkTitle: "Selenium Manager"
+weight: 11
+description: >
+    The Selenium Manager is a binary generated with Rust that manages driver installation.
+---
+
+## Applicability
+
+***Selenium bindings use this tool by default, so you do not need to download it or
+add anything to your code or do anything else to use it.***
+
+If, however, you need to access drivers with `curl` or a REST client,
+use one of the unofficial Selenium implementations,
+have special network requirements, or just want to better
+understand what the tool is doing, please continue reading.
+
+## Usage
+
+The Selenium bindings use the Manager automatically so long as the following conditions are met:
+* The driver location isn't specified in a `Service` class
+* A 3rd party driver manager is not installed
+* No drivers exist in directories included in the `PATH` Environment Variable
+
+To use it with Selenium Grid:
+* Start the grid with this additional argument: `--selenium-manager true`
+
+## Status
+
+This tool is still in beta, and Selenium is slowly adding support for its features.
+The current implementation is a "fall-back" option, which means it should only get used
+if the code execution would otherwise fail. So long as you specify the location of the
+driver in the appropriate [Service class]({{< ref "webdriver/drivers/service.md" >}})
+(or using System Properties in Java), the Selenium Manager will not be used.
+
+### History
+
+The first implementation of Selenium Manager was released in Selenium 4.6. We
+made an announcement about it [on our blog](https://www.selenium.dev/blog/2022/introducing-selenium-manager/).
+
+* Selenium 4.7 added support for IE Driver and improved error logging
+* Selenium 4.8 added support for beta and development versions of browsers
+* Selenium 4.9 added support for locating drivers based on provided location of browser binary 
+
+### Future
+
+We are tracking the various features as a [GitHub Project](https://github.com/orgs/SeleniumHQ/projects/5/views/1).
+The major features we are looking to support:
+* Proxies
+* Warning/fixing problems with drivers on `PATH`
+* Browser downloads based on `browserVersion` provided in capabilities
+
+## Configuration
+
+Specific values can be overridden by specifying environment variables or by using a config file 
+located by default at `~/.cache/selenium/selenium-manager-config.toml`.
+
+| CLI                              | Env Variable                                       | Config File                                         |
```

Comment:
Perhaps we could move this table into https://github.com/SeleniumHQ/selenium/tree/trunk/rust and then include a reference to the rust readme. That way as and when new stuff is added to the selenium manager, we wouldn't be required to remember to update this table also.

Reply:
I would rather see the README info moved into our docs where it can be translated. Things like the Roadmap and such or compilation issues, Rust weirdness, etc all belongs in the README. Everything a user might want to know should go in docs pages.